json_encoder_test.go

```diff
@@ -116,6 +116,10 @@ func TestJSONEncoderFields(t *testing.T) {
 		{"int", `"k\\":42`, func(e Encoder) { e.AddInt(`k\`, 42) }},
 		{"int64", `"k":42`, func(e Encoder) { e.AddInt64("k", 42) }},
 		{"int64", `"k\\":42`, func(e Encoder) { e.AddInt64(`k\`, 42) }},
+		{"uint", `"k":42`, func(e Encoder) { e.AddUInt("k", 42) }},
```

Comment:
suggest adding a test with a very large value `math.MaxUint64` so catch the overflow issue I mentioned earlier